templates/zerver/help/at-mention-a-team-member.md

```diff
@@ -3,12 +3,13 @@
 ![Mention a Team Member](/static/images/help/mention_user.png)
```

Comment:
The heading can probably be changed to `@-mention a team member`

Also, you need to remove the "(needs a note that you can't @mention when editing messages, since they may have already read the message / not clear how to notif them)" comment in `index.md` about this doc now that you've fixed it.